build-dependencies.just

```diff
@@ -27,6 +27,11 @@ install-mount-x86_64-unknown-linux-musl:
     sudo apt-get update
     sudo apt-get install -y fuse3 libfuse3-dev pkg-config
 
+# Install dependencies for the mount feature on aarch64-unknown-linux-musl
+install-mount-aarch64-unknown-linux-musl:
+    sudo apt-get update
+    sudo apt-get install -y fuse3 libfuse3-dev pkg-config
+
```

Comment:
Misunderstanding, `mount` won't be available due to musl cross-compilation. I meant a `install-default-aarch64-unknown-linux-musl:` one, for installing musl-tools.

Reply:
Is the `install-mount-x86_64-unknown-linux-musl:` also wrongly added?

Reply:
yes, please remove it! 🫂